### ISSUE TITLE
RetroDECK Compatibility

### ISSUE THREAD

**Issue by genefyx** [mmatyas/pegasus-frontend#1022]
RetroDECK compatibility would go a long way towards ease of use for Desktop & Steam Deck.

**Comment by mmatyas** [comment]
Hi! From a quick glance, RetroDeck seems to be a software bundle built on top of EmulationStation and RetroArch. Could you describe what sort of compatibility you'd wish to see from my side? If you'd like RetroDeck to include Pegasus, then you might want to ask the RetroDeck folks for that.

**Comment by mmatyas** [comment]
Ah, so the issue is that the Flatpak version doesn't detect the Steam games on a Steam Deck?

**Comment by genefyx** [comment]
No, to be clearer, RetroDeck works by itself on a steam deck but Pegasus isn't picking up the retro game details from the Emulation station inside the RetroDeck flatpak. As in games + scraped details.

**Comment by mmatyas** [comment]
Oh, I see; if RetroDeck is based on the EmulationStation-DE fork of EmulationStation, then this sounds like the duplicate of #1012. Pegasus only supports the official EmulationStation version, as the different ES forks are not compatible with each other.

**Comment by genefyx** [comment]
Do you think it won't ever be supported? Pulling the data && gamelist since from a flatpak would be a 1 time fix, (maybe a bash script?) but would work all OS's & all future game systems out of the box for the long foreseeable future.

**Comment by mmatyas** [comment]
It might be supported in the future, ES-DE seems to be a popular fork.

**Comment by genefyx** [comment]
Thank you so much for keeping it as an option! I'll keep an eye out for any updates.

**Comment by SecularSteve** [comment]
I myself recommend waiting it out until 2.0 is finally out of the pre-release stage, as that's the version of ES-DE they're trying to fully transition into.

**Comment by genefyx** [comment]
I'm unsure if waiting for 2.0 version was a roadblock but ES-DE 2.0 has recently released as of March 11, 2023. I hope this helps.
